diff --git a/Source/src/BitStream.cpp b/Source/src/BitStream.cpp
index f756d3aec..8eb8ef9e6 100644
--- a/Source/src/BitStream.cpp
+++ b/Source/src/BitStream.cpp
@@ -878,42 +878,7 @@ void BitStream::PrintBits( char *out ) const
 	out[strIndex++] = 0;
 }
 
-void BitStream::PrintBits( char *out, size_t outLength ) const
-{
-	if ( numberOfBitsUsed <= 0 )
-	{
-		strcpy_s(out, outLength, "No bits\n" );
-		return;
-	}
-
-	unsigned int strIndex=0;
-	for ( BitSize_t counter = 0; counter < BITS_TO_BYTES( numberOfBitsUsed ) && strIndex < 2000 ; counter++ )
-	{
-		BitSize_t stop;
-
-		if ( counter == ( numberOfBitsUsed - 1 ) >> 3 )
-			stop = 8 - ( ( ( numberOfBitsUsed - 1 ) & 7 ) + 1 );
-		else
-			stop = 0;
-
-		for ( BitSize_t counter2 = 7; counter2 >= stop; counter2-- )
-		{
-			if ( ( data[ counter ] >> counter2 ) & 1 )
-				out[strIndex++]='1';
-			else
-				out[strIndex++]='0';
 
-			if (counter2==0)
-				break;
-		}
-
-		out[strIndex++]=' ';
-	}
-
-	out[strIndex++]='\n';
-
-	out[strIndex++]=0;
-}
 void BitStream::PrintBits( void ) const
 {
 	char out[2048];
@@ -921,14 +886,6 @@ void BitStream::PrintBits( void ) const
 	RAKNET_DEBUG_PRINTF("%s", out);
 }
 
-void BitStream::PrintHex( char *out, size_t outLength ) const
-{
-	BitSize_t i;
-	for ( i=0; i < GetNumberOfBytesUsed(); i++)
-	{
-		sprintf_s(out+i*3, outLength-i*3, "%02x ", data[i]);
-	}
-}
 
 void BitStream::PrintHex( char *out ) const
 {
