diff --git a/include/SKSE/Version.h b/include/SKSE/Version.h
index 9a482353..666d6530 100644
--- a/include/SKSE/Version.h
+++ b/include/SKSE/Version.h
@@ -22,6 +22,7 @@ namespace SKSE
 	inline constexpr REL::Version RUNTIME_1_6_317(1, 6, 317, 0);
 	inline constexpr REL::Version RUNTIME_1_6_318(1, 6, 318, 0);
 	inline constexpr REL::Version RUNTIME_1_6_323(1, 6, 323, 0);
+	inline constexpr REL::Version RUNTIME_1_6_342(1, 6, 342, 0);
 
-	inline constexpr auto RUNTIME_LATEST = RUNTIME_1_6_323;
+	inline constexpr auto RUNTIME_LATEST = RUNTIME_1_6_342;
 }
