From 8e97bb6b6a663d8c6843006118510a41465d6a60 Mon Sep 17 00:00:00 2001
From: Leonid Pospelov <leonidpospelov.dev@gmail.com>
Date: Mon, 15 Apr 2024 22:13:03 +0500
Subject: [PATCH] Update TESObjectREFR.h

---
 include/RE/T/TESObjectREFR.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/RE/T/TESObjectREFR.h b/include/RE/T/TESObjectREFR.h
index bb0abfbfb..13c01de76 100644
--- a/include/RE/T/TESObjectREFR.h
+++ b/include/RE/T/TESObjectREFR.h
@@ -473,10 +473,11 @@ namespace RE
 		bool             preDestroyed;  // 93
 		std::uint32_t    pad94;         // 94
 
+		void              MoveTo_Impl(const ObjectRefHandle& a_targetHandle, TESObjectCELL* a_targetCell, TESWorldSpace* a_selfWorldSpace, const NiPoint3& a_position, const NiPoint3& a_rotation);
+
 	private:
 		InventoryChanges* ForceInitInventoryChanges();
 		InventoryChanges* MakeInventoryChanges();
-		void              MoveTo_Impl(const ObjectRefHandle& a_targetHandle, TESObjectCELL* a_targetCell, TESWorldSpace* a_selfWorldSpace, const NiPoint3& a_position, const NiPoint3& a_rotation);
 		void              PlayAnimation_Impl(NiControllerManager* a_manager, NiControllerSequence* a_toSeq, NiControllerSequence* a_fromSeq, bool a_arg4 = false);
 	};
 #ifndef SKYRIM_SUPPORT_AE
