diff --git a/include/SKSE/Impl/PCH.h b/include/SKSE/Impl/PCH.h
index e1c34290..ebe92cf7 100644
--- a/include/SKSE/Impl/PCH.h
+++ b/include/SKSE/Impl/PCH.h
@@ -56,6 +56,7 @@ static_assert(
 
 #pragma warning(push)
 #include <boost/stl_interfaces/iterator_interface.hpp>
+#include <fmt/format.h>
 #include <spdlog/spdlog.h>
 #pragma warning(pop)
 
diff --git a/include/SKSE/Logger.h b/include/SKSE/Logger.h
index 9bd62def..fa7218af 100644
--- a/include/SKSE/Logger.h
+++ b/include/SKSE/Logger.h
@@ -10,9 +10,8 @@
 	{                                                                     \
 		a_func() = delete;                                                \
                                                                           \
-		template <class T>                                                \
 		explicit a_func(                                                  \
-			T&& a_fmt,                                                    \
+			fmt::format_string<Args...> a_fmt,                            \
 			Args&&... a_args,                                             \
 			std::source_location a_loc = std::source_location::current()) \
 		{                                                                 \
@@ -22,13 +21,13 @@
 					static_cast<int>(a_loc.line()),                       \
 					a_loc.function_name() },                              \
 				spdlog::level::a_type,                                    \
-				std::forward<T>(a_fmt),                                   \
+				a_fmt,                                                    \
 				std::forward<Args>(a_args)...);                           \
 		}                                                                 \
 	};                                                                    \
                                                                           \
-	template <class T, class... Args>                                     \
-	a_func(T&&, Args&&...) -> a_func<Args...>;
+	template <class... Args>                                              \
+	a_func(fmt::format_string<Args...>, Args&&...) -> a_func<Args...>;
 
 namespace SKSE::log
 {
diff --git a/src/SKSE/Logger.cpp b/src/SKSE/Logger.cpp
index ca37209c..41241e02 100644
--- a/src/SKSE/Logger.cpp
+++ b/src/SKSE/Logger.cpp
@@ -29,16 +29,16 @@ namespace SKSE
 				if (a_event && a_event->errorMsg && std::regex_search(a_event->errorMsg, _filter)) {
 					switch (a_event->severity) {
 					case Severity::kInfo:
-						log::info(a_event->errorMsg);
+						log::info("{}"sv, a_event->errorMsg);
 						break;
 					case Severity::kWarning:
-						log::warn(a_event->errorMsg);
+						log::warn("{}"sv, a_event->errorMsg);
 						break;
 					case Severity::kError:
-						log::error(a_event->errorMsg);
+						log::error("{}"sv, a_event->errorMsg);
 						break;
 					case Severity::kFatal:
-						log::critical(a_event->errorMsg);
+						log::critical("{}"sv, a_event->errorMsg);
 						break;
 					}
 				}
