diff --git a/include/RE/E/ExtraDataList.h b/include/RE/E/ExtraDataList.h
index 320a71be..10e98d3b 100644
--- a/include/RE/E/ExtraDataList.h
+++ b/include/RE/E/ExtraDataList.h
@@ -156,7 +156,6 @@ namespace RE
 		void                  SetInventoryChanges(InventoryChanges* a_changes);
 		void                  SetOwner(TESForm* a_owner);
 
-	protected:
 		struct PresenceBitfield
 		{
 		public:
@@ -168,14 +167,15 @@ namespace RE
 		};
 		static_assert(sizeof(PresenceBitfield) == 0x18);
 
-		void MarkType(std::uint32_t a_type, bool a_cleared);
-		void MarkType(ExtraDataType a_type, bool a_cleared);
-
 		// members
 		BSExtraData*            _data;      // 00
 		PresenceBitfield*       _presence;  // 08
 		mutable BSReadWriteLock _lock;      // 10
 
+	protected:
+		void MarkType(std::uint32_t a_type, bool a_cleared);
+		void MarkType(ExtraDataType a_type, bool a_cleared);
+
 	private:
 		BSExtraData* GetByTypeImpl(ExtraDataType a_type) const;
 	};
